src/app/(account)/forgot-password/page.tsx

```diff
@@ -108,9 +108,9 @@ export default function ForgotPasswordPage() {
     });
 
     return (
-        <section className="flex justify-center">
-            <FancyRectangle colour="purple" offset="8" filled={true}>
-                <div className="z-0 w-[24rem] border-4 border-black bg-white px-8 py-8 text-black md:w-[32rem] md:px-12 md:py-12">
+        <section className="w-full max-w-lg">
+            <FancyRectangle colour="purple" offset="8" filled={true} fullWidth={true}>
```

Comment:
```suggestion
            <FancyRectangle colour="purple" offset="8" filled fullWidth>
```

Remove `={true}` can make the code cleaner (remove all, not only here)